### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/by/epam/ayem/module4/model/City.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package by.epam.ayem.module4.model;
+
+/*3. Создать объект класса Государство, используя классы Область, Район, Город. Методы:
+вывести на консоль столицу, количество областей, площадь, областные центры.*/
+
+public class City {
+
+    private String name;
+    private District district;
```
Comment (iyurinok) [inline src/by/epam/ayem/module4/model/City.java:12]: There is no getter for this private field. How are you going to access it. Also if the field is immutable (no setter it is better to mark it final)

Comment (iyurinok) [reply src/by/epam/ayem/module4/model/City.java:12]: This comment applies to all models

File: src/by/epam/ayem/module4/service/CountryService.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import java.util.Set;
+
+public class CountryService {
+
+    private Set<Country> countries;
+    private Set<Region> regions;
+    private Set<District> districts;
+    private List<City> cities;
+
+    public CountryService() {
```
Comment (iyurinok) [inline src/by/epam/ayem/module4/service/CountryService.java]: In general Service objects must be stateless and just take stateful objects as parameters.

File: src/by/epam/ayem/module4/service/CountryService.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+    public void addRegion(Region region) {
+        regions.add(region);
+    }
+
+    public void addCountry(Country country) {
+        countries.add(country);
+    }
+
+    public void countArea(Country country) {
```
Comment (iyurinok) [inline src/by/epam/ayem/module4/service/CountryService.java]: All method names should start from verb. So I would rename it to `showCountryArea`

File: src/by/epam/ayem/module4/service/CountryService.java [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        double area = 0;
+
+        for (Region region : regions) {
+            if (region.getCountry().getName().equalsIgnoreCase(country.getName())) ;
+            area += region.getArea();
+        }
+        System.out.printf("The country area = %.2f km2.\n", area);
+    }
+
+    public void regionsAmount(Country country) {
```
Comment (iyurinok) [inline src/by/epam/ayem/module4/service/CountryService.java]: Methods should begins with verb: set, get, print, show, run, do, etc...

File: src/by/epam/ayem/module4/service/CountryService.java [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+import by.epam.ayem.module4.model.Country;
+import by.epam.ayem.module4.model.District;
+import by.epam.ayem.module4.model.Region;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class CountryService {
```
Comment (iyurinok) [inline src/by/epam/ayem/module4/service/CountryService.java:14]: You should have javadoc for all public classes as well as public methods

Comment (iyurinok) [reply src/by/epam/ayem/module4/service/CountryService.java:14]: There are no unit tests. All classes with business logic must be covered with unit tests.

File: src/by/epam/ayem/module4/service/CountryService.java [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+    private Set<Country> countries;
+    private Set<Region> regions;
+    private Set<District> districts;
+    private List<City> cities;
+
+    public CountryService() {
+        this.countries = new HashSet<>();
+        this.regions = new HashSet<>();
+        this.districts = new HashSet<>();
+        this.cities = new ArrayList<>();
```
Comment (iyurinok) [inline src/by/epam/ayem/module4/service/CountryService.java]: If Service is stateless, to what model object these fields belongs to?

File: src/by/epam/ayem/module4/service/CountryService.java (hunk 5 above)
Comment (ghost) [reply src/by/epam/ayem/module4/service/CountryService.java:14]: Got it! I'm going to add them later, once I'm done fixing the actual code in all the tasks.

**Pull request merged**